src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -175,22 +171,39 @@ private string GetRepoUri()
         ///     quite expressive enough for the options
         /// </summary>
         /// <returns>True if they are being used properly, false otherwise.</returns>
-        private bool ValidateRootBuildOptions()
+        private bool ValidateRootBuildsOptions()
         {
+            bool hasNonIdRootBuildOptions =
+                (!string.IsNullOrEmpty(_options.RepoUri) ||
+                 !string.IsNullOrEmpty(_options.Channel) ||
+                 !string.IsNullOrEmpty(_options.Commit) ||
+                 _options.DownloadSdk ||
+                 _options.DownloadRuntime ||
+                 _options.DownloadAspNet);
+
             if (_options.RootBuildId != 0)
             {
-                if (!string.IsNullOrEmpty(_options.RepoUri) ||
-                  !string.IsNullOrEmpty(_options.Channel) ||
-                  !string.IsNullOrEmpty(_options.Commit) ||
-                  _options.DownloadSdk ||
-                  _options.DownloadRuntime ||
-                  _options.DownloadAspNet)
+                if (hasNonIdRootBuildOptions || _options.RootBuildIds.Any())
                 {
                     Console.WriteLine("--id should not be specified with other options.");
```

Comment:
Interesting. So the syntax will be `--id x` or `--ids x` for one, `--ids x,y,z` for many?

Reply:
Right. This was mainly about not removing an old option.  I am just going to change this to --id for many, even though it's not particularly good UX